Gemfile

```diff
@@ -1,3 +1,3 @@
 source "https://rubygems.org"
 
-gem 'github-pages'
\ No newline at end of file
+gem "jekyll-remote-theme", github: "benbalter/jekyll-remote-theme", branch: "master"
```

Comment:
The pages gem already includes this dependency, and is probably safer to use than pointing directly at the git branch